string pid
float32 remaining_burst_time
float64 timestamp

